redux_userlist_06/src/components/Dashboard/userProfile.tsx

```diff
@@ -1,21 +1,37 @@
+import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 
 import { UserDetails } from "../../interfaces";
 
 import "./styles.css";
 
-const UserProfile = (props: any)=> {
+const UserProfile = (props: any) => {
   const details: UserDetails = props.data;
+  const [visible,setVisible] = useState(false);
```

Comment:
ideally, boolean state should have prefix isVisible, hasMore ..... etc 
make it isVisible